VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -68,7 +73,22 @@ def device_list_for_billing(self):
     def get_total_used_size(self):
```

Comment:
are both of these pytjon file get_total_used_size() methods getting used ? can we remove duplicate methods ?

Reply:
Removed duplicate method in the new commit